src/annotator/highlighter.ts

```diff
@@ -54,7 +57,10 @@ function getPDFCanvas(highlightEl) {
  *   above the PDF.
  * @param {string} [cssClass] - CSS class(es) to add to the SVG highlight elements
  */
-function drawHighlightsAbovePDFCanvas(highlightEls, cssClass) {
+function drawHighlightsAbovePDFCanvas(
```

Comment:
The `@param`s above can be simplified by removing the types.

Reply:
Ah! Yes, that's a nice idea.